hal/halx86/apic/tsc.c

```diff
@@ -10,14 +10,14 @@
 
 #include <hal.h>
 #include "tsc.h"
+#include "apicp.h"
 #define NDEBUG
 #include <debug.h>
 
 LARGE_INTEGER HalpCpuClockFrequency = {{INITIAL_STALL_COUNT * 1000000}};
 
 UCHAR TscCalibrationPhase;
 ULONG64 TscCalibrationArray[NUM_SAMPLES];
-UCHAR HalpRtcClockVector = 0xD1;
 
 #define RTC_MODE 6 /* Mode 6 is 1024 Hz */
 #define SAMPLE_FREQENCY ((32768 << 1) >> RTC_MODE)
```

Comment:
```suggestion
#define SAMPLE_FREQUENCY ((32768 << 1) >> RTC_MODE)
```
+ typo fix in lines 47 and 75.

Reply:
OK, I fixed it, but please stop asking for unrelated changes. Especially if it's fucking typos.